### PR TITLE
Wmiinet util

### PR DESCRIPTION
**Pull request by rpetrusha**
# Unmanaged APIs for wminet-util.dll

## Summary

Documents the functions found in WMINet_Utils.dll. Preview pages are found at https://review.docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/wmi/?branch=pr-en-us-3601.
 
## Suggested Reviewers
//cc @mairaw, @AlexGhiondea, @danmosemsft

### CODE REVIEW

File: docs/framework/unmanaged-api/index.md [hunk 1]
```diff
@@ -59,6 +58,9 @@ This section includes information on unmanaged APIs that can be used by managed-
   
  [Strong Naming](../../../docs/framework/unmanaged-api/strong-naming/index.md)  
  Describes the strong naming API, which enables a client to administer strong name signing for assemblies.  
+
+[WMI and Performance Counters](wmi/index.md)
```
Comment (mairaw) [inline docs/framework/unmanaged-api/index.md]: nit: this is not aligned with the rest of the other entries.

File: docs/framework/unmanaged-api/toc.md [hunk 2]
```diff
@@ -10,6 +10,6 @@
 ## [Metadata](metadata/)
 ## [Profiling](profiling/)
 ## [Strong Naming](strong-naming/)
+## [WMI and Performance Counters](wmi/index.md)
```
Comment (mairaw) [inline docs/framework/unmanaged-api/toc.md]: shouldn't you link to the folder like the others?

File: docs/framework/unmanaged-api/wmi/toc.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# [ WMI and Performance Counters ](index.md)
+## [BeginEnumeration function]beginenumeration.md)
```
Comment (mairaw) [inline docs/framework/unmanaged-api/wmi/toc.md]: broken link

File: docs/framework/unmanaged-api/wmi/beginenumeration.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+`ptr`
+[in] A pointer to an [IWbemClassObject](https://msdn.microsoft.com/library/aa391433%28v=vs.85%29.aspx) instance.
+
+`lEnumFlags`  
+[in] A bitwise combination of the flags or values described in the [Remarks](#remarks) section that controls the properties included in the enumeration.
+
+## Return value
+
+The following values returned by this function are defined in the **WbemCli.h** header file, or you can define them as constants in your code:
```
Comment (mairaw) [inline docs/framework/unmanaged-api/wmi/beginenumeration.md]: nit: italic for file names?

File: includes/internalonly-unmanaged.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+
+[!NOTE] This API is for internal use only. It is not intended for use from developer code.
```
Comment (mairaw) [inline includes/internalonly-unmanaged.md]: shouldn't you style this as a proper note? also use contractions (It isn't) 

```
> [!NOTE]
> Text
```

File: docs/framework/unmanaged-api/wmi/beginenumeration.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
+---
+title: BeginEnumeration function
```
Comment (mairaw) [inline docs/framework/unmanaged-api/wmi/beginenumeration.md]: These titles will probably fall in the too short list. We might need to consider appending a value to the title metadata. Also, ideally you'd add the description too.

**Comment by rpetrusha** [comment]
@mairaw, I've addressed all of your comments. Thanks.

**Review by mairaw — APPROVED** [review]
Great job on getting this done @rpetrusha! Merging this since they need to go live soon.

**Pull request merged**